.github/workflows/vagrant-libvirt.yaml

```diff
@@ -0,0 +1,30 @@
+name: Build Garden Linux Vagrant Libvirt Image
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [ amd64, arm64 ]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup
+      run: |
+        sudo apt-get update
+        sudo env DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends qemu-user-static qemu-utils
+
+    - name: Build
+      run: |
+        ./build kvm-vagrant-${{ matrix.arch }}
+
+    - run: qemu-img convert -f raw -O qcow2  .build/kvm*-${{ matrix.arch }}-today-*.raw  gl.qcow2
+    - run: mv gl.qcow2 box.img; cp features/vagrant/Vagrantfile .; cp features/vagrant/metadata.json .
+    - run: tar cvzf gardenlinux_libvirt-${{ matrix.arch }}.box ./metadata.json ./Vagrantfile ./box.img
```

Comment:
Those steps could probably be done in the feature itself, meaning that it would be simpler to build a box locally...

Reply:
And I believe they should for the very reason you already stated.

Reply:
There is, you might want to have a look at how we create the Azure VHD images in https://github.com/gardenlinux/gardenlinux/blob/d910b0cb254c95a6cdf4596f3afa8ff591ad21b1/features/azure/info.yaml#L8-L10 and https://github.com/gardenlinux/gardenlinux/blob/main/features/azure/convert.vhd or the GCP .tar.gz images in https://github.com/gardenlinux/gardenlinux/blob/d910b0cb254c95a6cdf4596f3afa8ff591ad21b1/features/gcp/info.yaml#L6-L8 and https://github.com/gardenlinux/gardenlinux/blob/main/features/gcp/convert.gcpimage.tar.gz.